obelix/idefix/main.go

```diff
@@ -2,27 +2,61 @@ package main
 
 import (
 	"context"
-	"fmt"
-	"idefix/redis_management"
+	"idefix/devAuth"
+	"idefix/operator"
+	"idefix/producer"
+	"idefix/redis"
+	"idefix/watcher"
 	"log"
+	"time"
 )
 
-// Example for use RedisClient
-func example() {
-	rclient := redis_management.NewRedisClient(context.Background())
-
-	err := rclient.SetKey("1", "that my boy")
+func runGithub() {
+	kp, err := producer.New()
 	if err != nil {
-		log.Fatal(err)
+		log.Fatalf("failed to create new producer: %v", err)
+	}
+	rc := redis.NewClient(context.Background())
+	client := devAuth.GithubAuth()
+
+	w := watcher.Watcher{
+		Requester: client,
+		Operator: &operator.IdefixOperator{
+			RC: rc,
+			KP: kp,
+		},
+		Interval: 2 * time.Second,
```

Comment:
The interval should be an ENV variable.

Reply:
80e9339912b74be145bf08fd8fe739da9c4a0c65